src/currencyservice/package.json

```diff
@@ -20,7 +20,4 @@
     "request": "2.88.2",
     "xml2js": "0.4.23"
   },
-  "devDependencies": {
```

Comment:
I also see `"lint": "semistandard *.js"` line 8, does this need to be removed too?

Reply:
I'd assume so if we're removing the module entirely